src/vs/base/parts/quickopen/browser/quickOpenWidget.ts

```diff
@@ -411,7 +411,15 @@ export class QuickOpenWidget implements IModelProvider {
 		// Trigger open of element on selection
 		if (this.isVisible()) {
 			const context: IEntryRunContext = { event, keymods: this.extractKeyMods(event), quickNavigateConfiguration: this.quickNavigateConfiguration };
-			hide = this.model.runner.run(value, Mode.OPEN, context);
+			let mode = Mode.OPEN;
+
+			if (context.event instanceof KeyboardEvent) {
```

Comment:
how about we send over the `StandardKeyboardEvent` instead?  I'd prefer this handler to account for biz logic as to which mode to use; `elementSelected` being called in multiple locations.  `StandardKeyboardEvent` also has a `browserEvent` prop, so we can access the original event in this case.